backend/src/main/java/ai/verta/modeldb/artifactStore/storageservice/s3/S3Service.java

```diff
@@ -172,7 +172,17 @@ private synchronized void initializeS3ClientWithTemporaryCredentials(
             .withCredentials(new AWSStaticCredentialsProvider(awsCredentials))
             .withRegion(clientRegion)
             .build();
-    LOGGER.debug("s3client refreshed with temporary credentials");
+    try {
```

Comment:
after we refresh the credentials check if the new credentials actually work, if not then return false since the new s3Client is still invalid